htdocs/core/lib/functions.lib.php

```diff
@@ -7821,6 +7821,7 @@ function dol_htmlentitiesbr($stringtoencode, $nl2brmode = 0, $pagecodefrom = 'UT
 		if ($removelasteolbr) {
 			$newstring = preg_replace('/<br>$/i', '', $newstring); // Remove last <br> (remove only last one)
 		}
+		$newstring = preg_replace('/[\x{200B}-\x{200D}\x{FEFF}]/u', '', $newstring);
```

Comment:
This looks to be a range of several utf8 char.
Or is it just the zero width space char ?

Reply:
its for this char : 
\x{FEFF}    ZERO WIDTH NO-BREAK SPACE
\x{200B}    ZERO WIDTH SPACE
\x{200D}    ZERO WIDTH JOINER